src/test/java/uk/gov/pay/directdebit/webhook/gocardless/services/WebhookGoCardlessServiceTest.java

```diff
@@ -75,17 +82,19 @@ public void shouldStoreEventsWithAnUnhandledResource() {
     @Test
     public void shouldStoreButNotHandlePaymentEventsWithAnUnhandledAction() {
         GoCardlessEvent goCardlessEvent = aGoCardlessEventFixture().withResourceType(PAYMENTS).withAction("not_handled").toEntity();
-
         List<GoCardlessEvent> events = Collections.singletonList(goCardlessEvent);
+        
         webhookGoCardlessService.handleEvents(events);
         verify(mockedGoCardlessEventService).storeEvent(goCardlessEvent);
     }
 
     @Test
     public void shouldStoreButNotHandleMandateEventsWithAnUnhandledAction() {
         GoCardlessEvent goCardlessEvent = aGoCardlessEventFixture().withResourceType(MANDATES).withAction("not_handled_again").toEntity();
-
         List<GoCardlessEvent> events = Collections.singletonList(goCardlessEvent);
+
+        when(mockedMandateQueryService.findByGoCardlessMandateIdAndOrganisationId(any(), any())).thenReturn(mock(Mandate.class));
```

Comment:
We can do better than `any()` by explicitly creating the `goCardlessEvent` with a specific `GoCardlessMandateId` and a specific `GoCardlessOrganisationId`

Reply:
changed